force-app/main/default/lwc/registrationForm/registrationForm.html

```diff
@@ -0,0 +1,30 @@
+<template>
+    <lightning-card if:false={validMeetup} title="Invalid Meetup" class="slds-card__body_inner">
+        <p>This meetup can't be found, or doesn't exist. Please verify that you have the correct registration code URL.</p>
```

Comment:
Real world these text strings would probably be stored in Custom Labels or somewhere central for admin edits, translation workbench, etc.